src/ui-kit/components/actions/actions-dropdown/actions-dropdown.component.ts

```diff
@@ -48,17 +48,24 @@ export class SamActionsDropdownComponent {
   @Output() public emitCallback: EventEmitter<any> = new EventEmitter<any>();
 
   @ViewChildren('actionsList') public actionsList;
+
+  @ViewChild('actionButton') public actionButton;
+
   showActions = false;
   focusIndex = -1;
-  hideActions() {
+
+  hideActions(event) {
     return this.showActions = false;
   }
 
   toggleActions() {
-    if(!this.showActions){
+    this.showActions = !this.showActions;
+    if(this.showActions) {
+      this.setFocusOnFirstItem();
+    } else {
+      console.log("here reset focusIndex")
```

Comment:
@anuragmundada-reisys Can you remove the console log

Reply:
Add test cases for the new functionality.

Reply:
Can you also make sure cross-browser testing? It works all browsers.

Reply:
Done